jobs.go

```diff
@@ -188,9 +188,15 @@ func (s *JobsService) DownloadArtifactsFile(pid interface{}, refName string, job
 	if err != nil {
 		return nil, nil, err
 	}
-	u := fmt.Sprintf("projects/%s/jobs/artifacts/%s/download?job=%s", url.QueryEscape(project), refName, job)
 
-	req, err := s.client.NewRequest("GET", u, nil, options)
+    var q struct {
+        Job string `url:"job,omitempty" json:"job,omitempty"`
+    }
+    q.Job = job
```

Comment:
Please have a look at other option structs and use the same pattern here.

Reply:
@svanharmelen I think it's fine now. 
But now this commit include a breaking change as the job is no longer in the func parameters.

Tell me if i misunderstood something.